region/tourian/main.json

```diff
@@ -1244,7 +1243,7 @@
           },
           "locks": [
             {
-              "unlock": [ "DefeatMotherBrain" ]
+              "unlock": [ "ZebesSetAblaze" ]
```

Comment:
This changes nothing functionally, but for a rando you'd probably want to take away `ZebesSetAblaze`. Seems like that'd work out better if the door were logically locked behing `DefeatMotherBrain`.